### ISSUE TITLE
"Host" displayed on API Rules page is not correct

### ISSUE THREAD

**Issue by MingZhang-PS** [kyma-project/kyma#8642]
**Description**
The "Host" displayed on API Rules page doesn't have "Domain"
![image](https://user-images.githubusercontent.com/20200532/83227940-dc941200-a1b7-11ea-875e-89edd4df4c3f.png)
But if I press "edit" button, the "Domain" is auto populated
![image](https://user-images.githubusercontent.com/20200532/83227981-f46b9600-a1b7-11ea-851a-a76126a243da.png)

Kubernetes version: 1.15.11
cluster name: panda 
provider: GCP (Gardener)
Kyma version: 1.12.0
Browser: Chrome Version 81.0.4044.138 (Official Build) (64-bit) 
GCP VM Operation System: Ubuntu

**Expected result**
The "Host" shall be displayed exactly the same way in both display and edit mode.

**Actual result**
The "Host" displayed on API Rules page (display mode) is not correct, it is not a valid URL at all.

**Steps to reproduce**
The ApiRule CR underneath is created via command line as below
```
apiVersion: gateway.kyma-project.io/v1alpha1
kind: APIRule
metadata:
  {{- include "default_metadata" . | indent 2 }}
spec:
  gateway: kyma-gateway.kyma-system.svc.cluster.local
  service:
    name: ming-test-extension
    port: 80
    host: ming-test-extension-extension-playground
  rules:
    - path: /.*
      methods: ["GET", "POST", "PUT", "DELETE"]
      accessStrategies:
        - handler: noop
          config: {}
```

**Troubleshooting**

**Comment by kwiatekus** [comment]
Hi @MingZhang-PS 
This problem was already fixed by the following PR :
https://github.com/kyma-project/console/pull/1757

The fix will be part of 1.13 kyma release

**Comment by MingZhang-PS** [comment]
awesome!
